app/build.gradle

```diff
@@ -209,6 +219,12 @@ dependencies {
     // firebase
     implementation 'com.google.firebase:firebase-messaging-ktx:21.0.0'
     implementation 'com.google.firebase:firebase-analytics-ktx:18.0.0'
+    implementation 'com.google.android.gms:play-services-auth:19.0.0'
+    implementation platform('com.google.firebase:firebase-bom:26.8.0')
+    implementation 'com.google.firebase:firebase-auth-ktx:20.0.3'
+    implementation "androidx.browser:browser:1.3.0"
```

Comment:
// Import the BoM for the Firebase platform
    implementation platform('com.google.firebase:firebase-bom:26.8.0')

    // Declare the dependency for the Firebase Authentication library
    // When using the BoM, you don't specify versions in Firebase library dependencies
    implementation 'com.google.firebase:firebase-auth-ktx'